apps/app/pages/[workspaceSlug]/projects/[projectId]/settings/members.tsx

```diff
@@ -194,20 +202,27 @@ const MembersSettings: NextPage = () => {
                               className="absolute top-0 left-0 h-full w-full object-cover rounded-lg"
                             />
                           ) : (
-                            member.display_name.charAt(0)
+                            (member.display_name || member.email).charAt(0)
```

Comment:
In Case of Null value returned as in an edge case, how are we handling it.

Reply:
I assumed the email won't be null, but I added a null check to avoid an Exception Error.